src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -41,11 +39,14 @@ export function expandAliases(indicesOrAliases) {
   let ret = [].concat.apply([], indicesOrAliases);
   ret.sort();
   let last;
-  ret = $.map(ret, function (v) {
-    const r = last === v ? null : v;
-    last = v;
-    return r;
-  });
+  ret = ret.reduce((result, value) => {
+    if (last !== value) {
+      result.push(value);
+    }
+    last = value;
```

Comment:
nit: I'm not fully understand what happens here. I'm not a fan of retrieving and modifying any vars in parent scope of reduce function.  

If for some reason you need it, please use 3rd (index) and 4nd (array) arguments of reduce funciton